docs/rules/panel-datasource-rule.md

```diff
@@ -0,0 +1,4 @@
+# panel-datasource-rule
```

Comment:
Just a thought: You could combine all alike rules into one doc to consolidate this a bit. Something like:
panel-rules.md
- panel-datasource-rule
- panel-title-description-rule
- panel-units-rule

target-rules.md
-target-instance-rules
- etc.

template-rules.md
-template-datasource-rule
-etc.

Reply:
I like this, tho it breaks some of my automation for generating docs for each rule separately.

I'll give this some thought for a future enhancement/refactor PR. Thanks!